src/services/leafletLayerHelpers.js

```diff
@@ -55,6 +55,14 @@ angular.module('ui-leaflet')
                 return L.tileLayer(url, params.options);
             }
         },
```

Comment:
sorry @eczajk1 .. I hate how the code reviews chop of important parts of the code which I have to expand.. Anyway I should have looked at it locally more. My fault.